src/services/models/NFT.tsx

```diff
@@ -14,6 +14,6 @@ export interface NFT extends Dictionary {
   filename: string
   tags?: string[]
   royalty: number
-  tokenId?: string
+  tokenId?: string | number
```

Comment:
if it necessary and make block for you please choose type : any or another related type

>

Reply:
We don't use tokenId for now I can delete it as we use the nft ID.